staging_docs/admin/learn/authentication.md

```diff
@@ -117,11 +117,222 @@ via the setting `TOKEN_EXPIRATION_TIME`. The default expiration time is `300` se
 Below is provided an example of the settings file:
 
 ```python
-TOKEN_SERVER = "http://localhost:24817/token/"
+TOKEN_SERVER = "https://puffy.example.com/token/"
 TOKEN_SIGNATURE_ALGORITHM = 'ES256'
 PUBLIC_KEY_PATH = '/tmp/public_key.pem'
 PRIVATE_KEY_PATH = '/tmp/private_key.pem'
 ```
 
 Restart Pulp services in order to reload the updated settings. Pulp will fetch a domain for the token
 server and will initialize all handlers according to that.
+
+
+!!! note
+
+    Standard container tooling clients like podman, skopeo and docker handle token authentication calls on
+    the behalf of the user during pull and push operations, however, if you would like to debug registry
+    access or implement manual registry API calls, here are few examples on how token authentication works
+    behind the scene.
+
+Access the root registry endpoint to check if the token authentication was successfully configured and enabled:
+
+```bash
+    $ http 'https://puffy.example.com/v2/'
+
+    HTTP/1.1 401 Unauthorized
+    Allow: GET, HEAD, OPTIONS
+    Connection: close
+    Content-Length: 58
+    Content-Type: application/json
+    Date: Mon, 13 Jul 2020 09:56:54 GMT
+    Docker-Distribution-Api-Version: registry/2.0
+    Server: gunicorn/20.0.4
+    Vary: Accept
+    WWW-Authenticate: Bearer realm="https://puffy.example.com/token/",service="puffy.example.com"
+    X-Frame-Options: SAMEORIGIN
+
+    {
+        "detail": "Authentication credentials were not provided."
+    }
+```
+
+Since the request was not authenticated, registry returned 401 and in the authentication header there are
+details on how to authenticate. This time send the request with the specified realm and service:
+
+```bash
+$ http https://puffy.example.com/token/?service=puffy.example.com
+
+HTTP/1.1 200 OK
+Allow: GET, HEAD, OPTIONS
+Connection: close
+Content-Length: 609
+Content-Type: application/json
+Date: Mon, 13 Jul 2020 09:57:25 GMT
+Server: gunicorn/20.0.4
+Vary: Accept, Cookie
+X-Frame-Options: SAMEORIGIN
+
+{   
+    "access_token": <TOKEN>,
+    "expires_in": 300,
+    "issued_at": "2020-07-13T09:57:25.601760Z",
+    "token": <TOKEN>
+}
+```
+
+The token was generated to access registry root endpoint.
+
+```bash
+$ http --auth-type=jwt --auth=<TOKEN> https://puffy.example.com/v2/
+
+HTTP/1.1 200 OK
+Allow: GET, HEAD, OPTIONS
+Connection: close
+Content-Length: 2
+Content-Type: application/json
+Date: Mon, 13 Jul 2020 09:58:40 GMT
+Docker-Distribution-Api-Version: registry/2.0
+Server: gunicorn/20.0.4
+Vary: Accept
+X-Frame-Options: SAMEORIGIN
+
+{}
+```
+
+In order to access other registry endpoints, like manifests, blobs and tags the scope needs to be provided
+as well. In this context it will be a specific repository with the according action pull or push.
```

Comment:
```suggestion
In order to access other registry endpoints, like manifests, blobs and tags, the scope needs to be provided
as well. In this context, it will be a specific repository with the according action pull or push.
```